sympy/functions/special/zeta_functions.py

```diff
@@ -384,7 +385,19 @@ def _eval_nseries(self, x, n, logx, cdir=0):
                     s.append(term/k**nu)
                 return Add(*s) + o
 
-        return super(polylog, self)._eval_nseries(x, n, logx, cdir)
+        if z0 == S.One:
+            try:
+                _, exp = z.leadterm(x)
+            except (ValueError, NotImplementedError):
+                return self
+
+            if exp.is_nonnegative:
+                if (nu.is_integer and (1-nu).is_nonpositive) is False:
```

Comment:
Combining three valued bools with `and` does not work correctly in the `None` case. I have pointed to this page several times now:
https://docs.sympy.org/latest/guides/booleans.html